src/main/java/com/spotify/asyncdatastoreclient/Value.java

```diff
@@ -139,10 +143,157 @@ public Builder indexed(final boolean indexed) {
   }
 
   /**
+   * Create a new value containing a string.
+   *
+   * @param value The string to build a value from.
+   * @return A new value builder.
+   */
+  public static Value.Builder fromString(final String value) {
```

Comment:
Personally, I prefer method overloading in this scenario.

eg.
`public static Value.Builder from(final String value)`
`public static Value.Builder from(final boolean value)`
`public static Value.Builder from(final Date value)`

How do you feel about changing to that?